qovery/resource_container.go

```diff
@@ -552,8 +552,15 @@ func (r containerResource) Read(ctx context.Context, req resource.ReadRequest, r
 		return
 	}
 
+	// Hack to know if this method is triggered through an import
+	// EnvironmentID is always present except when importing the resource
+	var isTriggeredFromImport = false
```

Comment:
isTriggeredFromImport must be false on terraform import command?
Why the behavior have to be different for plan or apply command?

Reply:
`isTriggeredFromImport` is true on import command:
* on terraform import: we need to retrieve the advanced settings where default_value != adv_setting.value, otherwise you'll loose the advanced setting value that have been updated on UI side (that's why tests were failing on terraform import test)
* on terraform plan / apply: we need to ignore where default_value != adv_setting.value to not show the diff (the issue we have today)